src/dbus/applicationservice.cpp

```diff
@@ -182,7 +182,7 @@ QDBusObjectPath ApplicationService::Launch(const QString &action, const QStringL
             qsizetype location{0};
             location = commands.indexOf(R"(%f)");
             if (location != -1) {  // due to std::move, there only remove once
-                commands.remove(location);
+                commands.removeAt(location);
```

Comment:
are you sure?

https://doc.qt.io/qt-6/qlist.html#remove

it seems that remove(i) equals to removeAt(i),